src/github.js

```diff
@@ -94,10 +96,12 @@ const getSecurityVulnerabilitiesForPackage = async (token, packageName) => {
   })
   const vulnerabilities = data.securityVulnerabilities.nodes
   return vulnerabilities.map(vulnerability => ({
+    detailsUrl: vulnerability.advisory.permalink,
```

Comment:
are you confident `advisory` will always be here?

Reply:
I believe that's a safe assumption, yes. Based on [the docs](https://docs.github.com/en/free-pro-team@latest/graphql/reference/objects#securityvulnerability), it looks like the field is required (per the `!`).

Reply:
cool